metadata_backend/api/handlers/restapi.py

```diff
@@ -128,16 +128,26 @@ async def _get_data(self, req: Request) -> Dict:
             LOG.error(reason)
             raise web.HTTPBadRequest(reason=reason)
 
+    @staticmethod
+    async def _json_response(data: Union[Dict, List[Dict]]) -> Response:
```

Comment:
Note: we will probably need to use this in other sub-classes and make it more general to have optional headers (maybe an issue about :thinking: )

Reply:
https://github.com/CSCfi/metadata-submitter/issues/593